go.mod

```diff
@@ -113,6 +92,131 @@ require (
 	sigs.k8s.io/kind v0.11.1
 )
 
+require (
```

Comment:
Why do we have two require blocks now?

Reply:
Is this a new go 1.17 thing where all the indirect deps are in a new require area?

Reply:
that is a new 1.17 thing - second require block contains the indirect imports.